cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/PythonLanguage.kt

```diff
@@ -98,9 +98,9 @@ class PythonLanguage : Language<PythonLanguageFrontend>(), HasShortCircuitOperat
             ) {
                 // In Python, the // operation keeps the type as an int if both inputs are integers
                 // or casts it to a float otherwise.
-                return simpleTypes["int"]!!
+                return builtInTypes["int"]!!
```

Comment:
There is an (undocumented) function `getSimpleTypeOf` in `Language` that you used in other occasions for this.

Reply:
There is also this `newPrimitiveType` extension function which confuses me even more because it seems to return a type but then modifier its modifier?!

Reply:
Yes, the `newPrimitiveType` function is quite strange. It was used only in 3 places, so I just removed it.

I replaced the usage of `builtInTypes` with `getSimpleTypeOf`